Makefile

```diff
@@ -0,0 +1,19 @@
+IMAGE ?= prashcr/vim8s
+PLUGINS = test/plugins
+DOCKER = docker run -a stderr --rm -v $(PWD):/testplugin -v $(PWD)/test:/home "$(IMAGE)"
+
+$(PLUGINS)/vader.vim:
+	mkdir -p $(PLUGINS)
+	cd $(PLUGINS) && git clone https://github.com/junegunn/vader.vim.git
```

Comment:
Should we reference a particular commit ID so tests won't start failing if Vader updates?